modules/Bio/EnsEMBL/PipeConfig/BuildMart_conf.pm

```diff
@@ -77,6 +77,9 @@ sub pipeline_wide_parameters {
         mart_pass    => $self->o('pass'),
         mart_db_name => $self->o('mart'),
         tables_dir   => $self->o('tables_dir'),
+        scratch_dir  => catdir('/hps/nobackup/flicek/ensembl', $self->o('ENV', 'USER'), $self->o('pipeline_name')),
```

Comment:
I think these changes are unnecessary, given that this module inherits from GenericMart_conf, and these same dirs are defined there.

Reply:
you're right.